from app.exceptions.solana_tracker_base import DeFiTrackerProcessingException


class SolanaExternalNetworkException(DeFiTrackerProcessingException): pass
